ldaptive/src/main/java/org/ccci/idm/user/ldaptive/dao/LdaptiveUserDao.java

```diff
@@ -170,6 +171,7 @@ private int enqueueAllByFilter(@Nonnull final Collection<User> users, @Nullable
             conn.open();
             SearchOperation search = new SearchOperation(conn);
             final SearchRequest request = new SearchRequest(this.baseSearchDn, filter);
+            request.setReturnAttributes("*", LDAP_ATTR_PASSWORDCHANGEDTIME);
```

Comment:
Since we were not setting return attributes before, was the default to return all (i.e. '*'). But, the issue being that a return of 'all' attributes does not include `pwdChangedTime` unless explicitly specified?